administration/tests/test_admin_profiles.py

```diff
@@ -0,0 +1,93 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from administration.factories import (
+    AdminUserFactory,
+    AdminProfileFactory,
+)
+
+from utils.dump_response import dump  # noqa
+
+
+class TestAdminProfilesAPITests(APITestCase):
+    def setUp(self):
+        self.user = AdminUserFactory()
```

Comment:
I think we need some testcases for authenticated but not admin user

Reply:
Done